.changeset/rotten-tigers-lick.md

```diff
@@ -0,0 +1,5 @@
+---
+'@xstate/fsm': patch
+---
+
+state value added to transition type
```

Comment:
Hi @ksv90!

Changesets generate changelog entries for users so you want to _think like a user_ when writing your changesets.

If you're a user reading a changelog entry, you'd want to know things like:

- **What** is the change?
- **Why** is the change important?
- **How** do I make use of the change?

Your changeset isn't _technically wrong_, it just doesn't help users understand the change that you've made here!